framework/tests/repeatable-qualifiers/Issue1307.java

```diff
@@ -1,15 +0,0 @@
-// Test case for Issue 1307
```

Comment:
Rather than deleting the file, [my suggestion](https://github.com/typetools/checker-framework/pull/2350#discussion_r265666828) was to move it into a different directory where it will be tested.

Reply:
Oh, I thought that the test case was too wrong.